internal/db/db.go

```diff
@@ -34,9 +35,23 @@ func StringToDbType(dialect string) (DbType, error) {
 	}
 }
 
+// DB is a wrapper around the ORM
+type DB struct {
+	*gorm.DB
+}
+
+// SqlDB returns the underlying driver.
+//
+// Note: This func is not named DB(), because our chosen of ORM (gorm) which is
+// embedded for various reasons, already exports a DB() func and it's base type
+// is also an export DB type
+func (d *DB) SqlDB() (*sql.DB, error) {
+	return d.DB.DB()
```

Comment:
I think this should check if `d.DB` is `nil` and return an error if it is.

Reply:
ty!